src/main/kotlin/app/revanced/patches/tiktok/interaction/downloads/fingerprints/ACLCommonShareFingerprint2.kt

```diff
@@ -1,13 +1,10 @@
 package app.revanced.patches.tiktok.interaction.downloads.fingerprints
 
-import app.revanced.patcher.annotation.Name
 import app.revanced.patcher.extensions.or
 import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
-import app.revanced.patches.tiktok.interaction.downloads.annotations.DownloadsCompatibility
 import com.android.tools.smali.dexlib2.AccessFlags
 
-@Name("Acl common share get show type")
-@DownloadsCompatibility
+
```

Comment:
❓ : Too many space between import and object
> The rest of the file use one space to separate between import and object, we should follow the same here by using one instead of two spaces.

```suggestion
```